cmd/althea/cmd/gentx.go

```diff
@@ -1,537 +0,0 @@
-package cmd
```

Comment:
Our gentx (once all the Gravity specifics were removed) was functionally identical to the SDK one, so I removed it.